src/transaction/locator_sr.c

```diff
@@ -13937,6 +13941,166 @@ has_errors_filtered_for_insert (std::vector<int> error_filter_array)
 }
 // *INDENT-ON*
 
+
+static inline void
+locator_initialize_classname_entry (LOCATOR_CLASSNAME_ENTRY * entry, const char *classname, const OID * classoid)
```

Comment:
Can't this be used in `locator_initialize` as well?

Reply:
Yes, it can be used in `locator_initialize ()` as well if `entry->e_name` is copied by `strdup ()` as you suggested, 
but I didn't replace the code in `locator_initialize()` because it is not related to this issue.